graphql_compiler/cost_estimation/statistics.py

```diff
@@ -24,7 +24,7 @@ def __repr__(self):
 
     @abstractmethod
     def get_class_count(self, class_name):
-        """Return how many objects have, or inherit, the given class name.
+        """Return how many object instances have, or inherit, the given class name.
```

Comment:
nit: How about "vertex or edge instances have"? Everything is an object in python.